### ISSUE TITLE
Resolve DOI to content

### ISSUE THREAD

**Issue by mfenner** [datacite/datacite#53]
As a data repository, I want to enable direct access to my data, to enable automated workflows.

**Comment by espacial** [comment]
Are we talking about content resolution/negotiation here?

**Comment by mfenner** [comment]
Yes, I think so. I think this can be resolved with proper documentation for now.
